### DISCUSSION TITLE
Whisper in a web app AZURE

### DISCUSSION THREAD

**Discussion by ockdavid** [openai/whisper#2095] [Show and tell]
Hi! I'm trying to deploy a web app in Azure but every time that the requirements include the whisper openai library it doesn't deploy. Does anyone know why is this happening?

**Comment by Suyash-Muley** [comment]
@ockdavid  Did you find any solution for this? As even we have observed our webapp failing deployment when it tries to install the model

**Comment by ockdavid** [comment]
Yes! There were some libraries in the requirements file that didn't allow the app to deploy in Azure because they were compatible only in Windows. How did I figure it out this? I installed one by one the libraries so I could detect after the installation of which one fails the deployment. However, when you install a library, by default, more libraries are installed. So the problem was some of the libraries installed by default, not by the one I needed.

**Reply by Suyash-Muley** [reply]
Got it. I had removed it temporarily to bypass the failures I'll; be looking at this issue more.
Thanks!!
